src/backend/SyncBackendConnection.cpp

```diff
@@ -265,39 +268,47 @@ bool SyncBackendConnection::update_dds_model(
         EntityKind::PARTICIPANT,
         id,
         &SyncBackendConnection::update_participant_item,
-        &SyncBackendConnection::create_participant_data_);
+        &SyncBackendConnection::create_participant_data_,
+        inactive_visible);
 }
 
 bool SyncBackendConnection::update_get_data_dialog_entity_id(
         models::ListModel* entity_model,
-        EntityKind entity_kind)
+        EntityKind entity_kind,
+        bool inactive_visible /* true */)
 {
     bool changed = false;
 
-    try
+    for (auto entity_id : get_entities(entity_kind, ID_ALL))
```

Comment:
Why have you removed the try catch here?

Reply:
The main idea (separating the files was supposed to help with that) is to encapsulate each `StatisticsBackend` method in a SyncBackendConnection method.
The methods used by this function are from `SyncBackendConnection` and not from `StatisticsBackend`. Thus, they will never raise an exception.